public/static/docs/command-reference/import.md

```diff
@@ -159,15 +159,10 @@ deps:
       rev_lock: 0547f5883fb18e523e35578e2f0d19648c8f2d5c
 ```
 
-If the
-[Git revision](https://git-scm.com/book/en/v2/Git-Internals-Git-References)
-moves (e.g. a branch), you may use `dvc update` to bring the data up to date.
-However, for typically static references (e.g. tags), or for SHA commits, in
-order to actually "update" an import, it's necessary to **re-import the data**
-instead, by using `dvc import` again without or with a different `--rev`. This
-will overwrite the import stage (DVC-file), either removing or replacing the
-`rev` field, respectively. This can produce an import stage that is able to be
-updated normally with `dvc update` going forward. For example:
+If `rev` moves (e.g. a branch), you may use `dvc update` to bring the data up to
+date. If it's static however (e.g. a commit hash), it would be necessary to
+**re-import the data** instead, by using `dvc import` again with a different (or
+without) `--rev`. For example:
```

Comment:
Here's the simplification I was talking about in https://github.com/iterative/dvc.org/pull/933#discussion_r373742627 @shcheklein

Reply:
I assume this one is good 🙂

Reply:
`moves` is not clear enough. How about something like this:


If `rev` is a branch or tag name (any other <right git term>) it is usual that the most recent commit in the brach has updated, or tag now points to a different commit, etc. To update data and the DVC-file to point to the new commit in the branch or tag (<mention rev-lock?>) use ...

If `rev` is static `dvc update` does not have effect, you would need to re-import a new commit (or something like this).

Reply:
OK. How about the newest version? (b0de4099)

```markdown
If `rev` is a branch, it will probably have newer commits periodically. To bring
the data up to date (and update `rev_lock` in the DVC-file), simply use
`dvc update`. If `rev` is more static (e.g. a tag or a commit hash) though, you
would need to **re-import** a different commit instead by using `dvc import`
again with a different (or without) `--rev`. For example:
```

Reply:
so, `dvc update` works with tags as well, it not that rate that people use them (like `latest` to designate the last to date model file).

otherwise looks good. I'm not sure if there is "static" term used in Git docs.

Reply:
> dvc update works with tags as well... not that rare that people use them (like latest

I would argue it's pretty rare but OK, I came up with another version that is more complete, hopefully not overcomplicated again:

```markdown
If `rev` is a Git branch or tag, it may have new commits periodically. To bring
the data up to date (and update `rev_lock` in the DVC-file), simply use
`dvc update <stage>.dvc`. If `rev` is a specific commit (does not change),
`dvc update` will never have an effect on this import stage. You may
**re-import** a different commit instead, by using `dvc import` again with a
different (or without) `--rev`. For example:
```

Note that there's a feature request to add `--rev` to `dvc update` but it's not implemented yet: iterative/dvc/issues/2849 – we'll review this text again when that is implemented.

> not sure if there is "static" term used in Git docs

I don't know. I was just using general language 🙂 but that word is gone now anyway.

Reply:
`it may have new commits periodically` - tags may point to the new commit, not have them. Otherwise looks good :)

Reply:
Changed to "it may point to a newer commit periodically" but it feels to technical or Git-specific again... ⏳

Reply:
How about

```markdown
If `rev` is a Git branch or tag (where the commit it points to changes), the
data source may have updates at a later time. To bring it up to date if so (and
update `rev_lock` in the DVC-file), simply use `dvc update <stage>.dvc`. If
`rev` is a specific commit (does not change), `dvc update` will never have an
effect on the import stage. You may **re-import** a different commit instead, by
using `dvc import` again with a different (or without) `--rev`. For example:
```

(latest version in f409d81b)

Reply:
👍